app.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const app = express();
+
+app.get("/bugs", (request, response) => {
+    response.send("<h1>99 little bugs in the code</h1><br><a href=localhost:8888/bugs/101>pull one down, patch it around</a>")
+})
+
+app.get("/bugs/101", (req,res) => {
```

Comment:
here you want to use a parameter because `101` is hardcoded. so it would look like `/bugs/:numOfBugs' then you would destructure it below using `const {numOfBugs} = req.params`